internal/persistence/persistors/mysqlstore/store_category_test.go

```diff
@@ -338,15 +338,15 @@ func Test_UpdateCategories_Fail(t *testing.T) {
 	assert.Nil(t, cat, "unexpected non nil entry")
 }
 
-type testCaseRestoreCategory struct {
+type restoreCategoryTestCase struct {
```

Comment:
lets update all testcase structs to have the format of testCase__WHATEVER__

so.. **testCaseRestoreCategory** is the one that is actually _correct_